server/tso/local_allocator.go

```diff
@@ -99,8 +96,8 @@ func (lta *LocalTSOAllocator) SetTSO(tso uint64) error {
 // GenerateTSO is used to generate a given number of TSOs.
 // Make sure you have initialized the TSO allocator before calling.
 func (lta *LocalTSOAllocator) GenerateTSO(count uint32) (pdpb.Timestamp, error) {
-	// Todo: use the low bits of TSO's logical part to distinguish the different local TSO
-	return lta.timestampOracle.getTS(lta.leadership, count)
+	return lta.timestampOracle.getTS(
+		lta.leadership, count, lta.allocatorManager.GetClusterDCLocationsNumber())
```

Comment:
Should get the suffix of the dc locator? this is the total number?

Reply:
This is the total number.

Reply:
So, Is the suffixBits dynamically change?

Reply:
Yes, it will be calculated by the `tso.CalSuffixBits` method according to the given `dc-location` number.